### DISCUSSION TITLE
Setting the frequency command of HiTechnic Color Sensor V2

### DISCUSSION THREAD

**Discussion by dianabarrett** [pybricks/support#346] [Q&A] [answered]
Hi!

Is there a way of setting 50Hz or 60Hz frequency values of the HT Color Sensor V2?
I've seen a similar topic here on ev3dev discussion board https://github.com/ev3dev/ev3dev/issues/1292#issuecomment-508521846 and an answer by @dlech. I've tried something like this, gathered from examples and answers:
```python
class HiTechColorSensor(Ev3devSensor):
    def __init__(self, port):
        # Initialize the parent class.
        super().__init__(port)

        # Get the sysfs path.
        self.path = '/sys/class/lego-sensor/sensor' + str(self.sensor_index)

    # configure the sensor for 60Hz
    def set_60hz(self):
        with open(self.path + 'command', 'w') as file:
            file.write('60Hz')
```
I'm getting an error when calling `set_60Hz`:
### **`OSError: [Errno 13] EACCES`**

**Question:**
Is it at all possible to set the frequency to a specific value? If yes, what would be the approach?

**Comment by dlech** [comment, marked as answer]
> ```python
> self.path = '/sys/class/lego-sensor/sensor' + str(self.sensor_index)
> ```

This needs a trailing slash:

```python
self.path = '/sys/class/lego-sensor/sensor' + str(self.sensor_index) + '/'
```

**Reply by dianabarrett** [reply]
Thank you, @dlech. I've added the slash and now I'm getting another error:
### `OSError: [Errno 22] EINVAL`

**Reply by dlech** [reply]
I think `60HZ` needs to be all upper case.

**Reply by dianabarrett** [reply]
That was it! Thank you very much!!
